src/ray/gcs/tables.h

```diff
@@ -481,10 +497,11 @@ class ErrorTable : private Log<JobID, ErrorTableData> {
                            const std::string &error_message, double timestamp);
 };
 
-class ProfileTable : private Log<UniqueID, ProfileTableData> {
+class ProfileTable : public Log<UniqueID, ProfileTableData> {
```

Comment:
Why the change to `public`?

Reply:
Originally we want to use the AddShards function. Change reverted.